source/ed/connectors/fusio_parser.h

```diff
@@ -32,7 +32,7 @@ www.navitia.io
 #include "gtfs_parser.h"
 
 /**
- * Read CanalTP custom transportation files
+ * Read Hove custom transportation files
```

Comment:
Maybe adding a link to https://github.com/hove-io/ntfs-specification would be pertinent :)

Reply:
Good idea. Done in 2c6b89133.